proposals/lip-0048.md

```diff
@@ -78,43 +68,37 @@ The Fee module does not expose any functions.
 
 #### Verify Transaction
 
-During `trs` verification, the following logic is applied:
-
 ```python
-minFee = MIN_FEE_PER_BYTE * size(trs)
-         + extraCommandFee(trs.moduleID, trs.commandID)
-if trs.fee < minFee:
-     trs is invalid
+def verify(trs: Transaction) -> None:
+    minFee = MIN_FEE_PER_BYTE * size(trs)
```

Comment:
```suggestion
    trsSize = len(encodeTransaction(trs)) # encodeTransaction is defined in LIP "Define new transaction schema"
    minFee = MIN_FEE_PER_BYTE * trsSize
```

Reply:
I would be completely precise since we now have the correct API. Same later on in other functions

Reply:
Changed to `encodeTransaction`.